code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -947,6 +961,7 @@ datum/reagent/medicine/styptic_powder/overdose_start(mob/living/M)
 /datum/reagent/medicine/neurine
 	name = "Neurine"
 	id = "neurine"
+	value = 6
```

Comment:
but it's easy to make in absolutely massive quantities (I've literally never managed to make less than 100 at a time) since it's just oil, welding fuel, oxygenx2, hydrogen, water, sugar (oil can be made in upgraded chem dispensers)

Reply:
Got a suggestion for a price then?